subsys/bluetooth/host/l2cap.c

```diff
@@ -3068,6 +3070,20 @@ int bt_l2cap_chan_disconnect(struct bt_l2cap_chan *chan)
 	return 0;
 }
 
+__maybe_unused static bool user_data_not_empty(const struct net_buf *buf)
+{
+	size_t ud_len = sizeof(struct closure);
+	uint8_t *ud = net_buf_user_data(buf);
+
+	for (size_t i; i < ud_len; i++) {
```

Comment:
```suggestion
	for (size_t i = 0U; i < ud_len; i++) {
```

Reply:
dont know what you mean, it works on my machine 🤪